polaris-react/src/components/ContentBlock/ContentBlock.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import type {ComponentMeta} from '@storybook/react';
+import {ContentBlock, Box, Text} from '@shopify/polaris';
+
+export default {
+  component: ContentBlock,
+} as ComponentMeta<typeof ContentBlock>;
+
+const placeHolder = {
+  background: 'var(--p-background-selected)',
+  borderRadius: 'var(--p-border-radius-05)',
+  border: '1px solid var(--p-surface-dark)',
+  padding: 'var(--p-space-4)',
+  width: '100%',
+  height: 'var(--p-space-32)',
+};
+
+export function Medium() {
+  return (
+    <ContentBlock width="medium">
```

Comment:
If we update `Width` type in `ContentBlock`, this needs to be updated
```suggestion
    <ContentBlock width="md">
```